resources/js/blocks.js

```diff
@@ -327,6 +327,12 @@ SCRIPTING_AREA.addEventListener('contextmenu', function(ev) {
     }
 });
 
+SCRIPTING_AREA.addEventListener('click', function(ev) {
```

Comment:
(I needed to figure out how to pull from upstream, I think I got that figured out)